packages/react/test/specs/components/Accordion/Accordion-test.tsx

```diff
@@ -203,6 +203,21 @@ describe('Accordion', () => {
 
       expect(onTitleClick).toBeCalledTimes(1)
     })
+
+    it('when clicked on disabled panel, do not call onClick and onTitleClick if provided by the user', () => {
```

Comment:
should we test for keyboard enter/space, or is it not focusable if it is disabled?